### PR TITLE
docs: Added the docs for getting started with tests

### PR DESCRIPTION
**Pull request by vatsashah45**
GitHub Issue (If applicable): closes #

#2248 

## PR Type

What kind of change does this PR introduce?

## What is the new behavior?

Added the docs for getting started with tests

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md). (for bug fixes / features) 
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/GettingStartedWithTests.md
Comment (eriklimakc) [inline doc/GettingStartedWithTests.md:1]: We also need the following as in https://platform.uno/docs/articles/external/uno.extensions/doc/GettingStarted.html, but you need to adjust things, like the command to create a solution with tests and server:

1. Installing extension templates

2. Exploring the Solution

Comment (agneszitte) [reply doc/GettingStartedWithTests.md:1]: -> see my comment on the other PR https://github.com/unoplatform/uno.extensions/pull/2249/files#r1563150841

Comment (jeromelaban) [reply doc/GettingStartedWithTests.md:1]: Adding this new page also requires a reference in the TOC.

**Comment by eriklimakc** [comment]
I cherry-picked commits from this PR to #2249.

**Pull request closed without merging**